afm-backend/controller/userController.py

```diff
@@ -22,4 +23,25 @@ def login():
     db.commit()    
     access_token = create_access_token(identity=username, expires_delta=datetime.timedelta(minutes=60))
     db = None
-    return jsonify(access_token=access_token)
\ No newline at end of file
+    return jsonify(access_token=access_token)
+
+@user_blueprint.route("/reset_password", methods=["PUT"])
+@jwt_required()
+def reset_password():
+    db = Database()
+    cursor = db.cursor()
+    username = get_jwt_identity()
+    password = request.json.get("password", None)
+    confirm_password = request.json.get("confirm_password", None)
```

Comment:
Change confirm_password to confirmPassword
To keep it consistent with other APIs

Reply:
Got it